.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": [
+    "airbnb-base",
+    "prettier"
+  ],
+  "plugins": [
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": [
+      "error",
+      {
+        "singleQuote": true,
+        "semi": false,
+        "parser": "flow",
+        "trailingComma": "es5"
+      }
+    ],
+    "no-console": 0
+  },
+  "globals" : {
+    "it": false,
+    "expect": false,
+    "describe": false,
+    "jest": false
```

Comment:
Have you used this? https://www.npmjs.com/package/eslint-plugin-jest

Reply:
Or this: https://stackoverflow.com/a/40265356/3055448

Reply:
Didn't know about them :man_facepalming: 

Will use the latter